packages/rule-components/src/ReportDetails/Markdown.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import doT from 'dot';
+import marked from 'marked';
+import sanitizeHtml from 'sanitize-html';
+
+marked.Renderer.prototype.link = (href, title, text) => {
+    return `<a href="${href}" rel="noopener noreferrer" target="_blank" className="ins-c-rule__link-in-description">${text}</a>`;
+};
+
+marked.setOptions({
+    renderer: new marked.Renderer()
+});
+
+const Markdown = ({ template, definitions }) => {
+    const DOT_SETTINGS = { ...doT.templateSettings, varname: [ 'pydata' ], strip: false };
+    const sanitizeOptions = {
+        allowedAttributes: { ...sanitizeHtml.defaults.allowedAttributes, '*': [ 'style' ] },
+        transformTags: {
+            ul(tagName, attribs) {
+                return {
+                    tagName: 'ul',
+                    attribs: { class: 'pf-c-list' }
+                };
+            }
+        },
+        textFilter(text) {
+            return text.replace(/&amp;/g, '&').replace(/&gt;/g, '>').replace(/&lt;/g, '<');
+        }
+    };
+    const externalLinkIcon = '<i class="fas fa-external-link-alt"></i>';
+
+    try {
+        const compiledDot = definitions ? doT.template(template, DOT_SETTINGS)(definitions) : template;
+        const compiledMd = marked(sanitizeHtml(compiledDot, sanitizeOptions));
+
+        return <div dangerouslySetInnerHTML={{
+            __html: compiledMd
+            .replace(/<ul>/gim, `<ul class="pf-c-list" style="font-size: inherit">`)
+            .replace(/<a>/gim, `<a rel="noopener noreferrer" target="_blank">`)
```

Comment:
There's no href in here. Is that intentional?

Reply:
I copied it from `inventory-insights`, so I think it should be like this 😃